<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>varDialog</class>
 <widget class="QDialog" name="varDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>395</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>395</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_4" rowstretch="1,1,1" columnstretch="5,0,5">
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="verticalSpacing">
        <number>2</number>
       </property>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="line_bottom_right_x">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="line_bottom_right_y">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Bottom-Right:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout" rowminimumheight="1,2,2">
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>2</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="line_top_left_x">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Top-Left:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="line_top_left_y">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3,2">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Log: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="line_log">
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_format">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <item>
          <property name="text">
           <string>px</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>m</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>deg</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Function: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_type">
         <item>
          <property name="text">
           <string>Variogram</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Covariance</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Abort|QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>varDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
